app/models/katello/candlepin/repository_mapper.rb

```diff
@@ -44,7 +44,7 @@ def build_repository
       end
 
       def validate!
-        return if katello_content_type == Repository::OSTREE_TYPE
+        return if katello_content_type == Repository::OSTREE_TYPE || Setting[:content_disconnected]
```

Comment:
The repository mapper should have a test which asserts "validate! does not issue any http requests when in disconnected mode"

Reply:
Agree. That should be simple to add

Reply:
Sounds good, will add a test for this scenario